README.md

```diff
@@ -1,4 +1,6 @@
 [![CircleCI](https://circleci.com/gh/LD4P/sinopia_indexing_pipeline.svg?style=svg)](https://circleci.com/gh/LD4P/sinopia_indexing_pipeline)
+[![Code Climate](https://codeclimate.com/github/LD4P/sinopia_indexing_pipeline/badges/gpa.svg)](https://codeclimate.com/github/LD4P/sinopia_indexing_pipeline)
```

Comment:
I'm not familiar with Code Climate - I'll be interested to see how the **trends** metrics develop over time. Do know if we add other projects like the `sinopia_editor` that it would look and generate these trends based on it's commit history?

Reply:
@jermnelson IIRC, the trends are based on builds for a project over time not the commit history, alas.